src/api/app/views/webui/users/_breadcrumb_items.html.haml

```diff
@@ -8,4 +8,10 @@
       Sign Up
   - when 'show'
     %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Home of #{@displayed_user}
\ No newline at end of file
+      Home of #{@displayed_user}
+  - when 'edit'
+    = render partial: 'webui/configuration/breadcrumb_items'
```

Comment:
There are 2 houses now, it must be because we render this partial and the one on line 1.

![2-houses](https://user-images.githubusercontent.com/1102934/65021035-a155f980-d92e-11e9-8062-90000303e1b5.png)